src/d3a/models/market/blockchain_interface.py

```diff
@@ -19,8 +19,21 @@
 from d3a.events.event_structures import MarketEvent
 from d3a.d3a_core.exceptions import InvalidTrade
 from d3a_interface.constants_limits import ConstSettings, GlobalConfig
+from d3a.models.market.blockchain_utils import get_function_metadata, \
+    address_to_hex, swap_byte_order, BOB_ADDRESS, ALICE_ADDRESS, \
+    test_value, test_rate, main_address, mnemonic, hex2, default_url
+from substrateinterface import SubstrateInterface, SubstrateRequestException, Keypair
+from pathlib import Path
 import platform
+import random
+import logging
 
+logging.basicConfig(filename="trades.log",
+                    format='%(asctime)s %(message)s',
+                    filemode='w')
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
```

Comment:
Base logger should be already configured, you can just `import logging` and use `logging.info` etc.